templates/CRM/Contact/Form/Selector.tpl

```diff
@@ -123,7 +119,8 @@
     {/foreach}
   {/if}
 </table>
-
+{include file="CRM/common/pagerAToZ.tpl"}
```

Comment:
I'd suggest moving this line to just after  line 10 `{include file="CRM/common/pager.tpl" location="top"}`

That will restore the Alpha selector where I believe it is needed most.   

The rest of these changes are an improvement, this one is unexpected.